### PR TITLE
Added snpEff-4.3k-Java-1.7.0_80.eb based on snpEff-4.1d-Java-1.7.0_80.eb

### PR DESCRIPTION
**Pull request by danielnavarrogomez**

### CODE REVIEW

**Comment by boegel** [comment]
@danielnavarrogomez I changed the target branch to `develop`, since PRs are not merged directly into `master`. For future contributions, I recommend taking a look at `--new-pr`, cfr. http://easybuild.readthedocs.io/en/latest/Integration_with_GitHub.html#github-new-pr

File: easybuild/easyconfigs/s/snpEff/snpEff-4.3k-Java-1.7.0_80.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+homepage = 'http://sourceforge.net/projects/snpeff/'
+description = """SnpEff is a variant annotation and effect prediction tool. 
+ It annotates and predicts the effects of genetic variants (such as amino acid changes)."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+java = 'Java'
+javaver = '1.7.0_80'
+versionsuffix = '-%s-%s' % (java, javaver)
```
Comment (boegel) [inline easybuild/easyconfigs/s/snpEff/snpEff-4.3k-Java-1.7.0_80.eb]: @danielnavarrogomez We have a better way of doing this now for a while, using the `%(javaver)s` template, see also http://easybuild.readthedocs.io/en/latest/version-specific/easyconfig_templates.html#avail-easyconfig-templates

I made this changed consistently in https://github.com/danielnavarrogomez/easybuild-easyconfigs/pull/1; if you merge that pull request, this one will be updated accordingly.

Comment (boegel) [reply easybuild/easyconfigs/s/snpEff/snpEff-4.3k-Java-1.7.0_80.eb]: @danielnavarrogomez ping?

**Comment by migueldiascosta** [comment]
@danielnavarrogomez, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**